AU/Public/Update-Package.ps1

```diff
@@ -224,8 +224,11 @@ function Update-Package {
         $script:is_forced = $false
         if ([AUVersion] $Latest.Version -gt [AUVersion] $Latest.NuspecVersion) {
             if (!($NoCheckChocoVersion -or $Force)) {
-                if ( !$au_GalleryUrl ) { $au_GalleryUrl = 'https://chocolatey.org' } 
-                $choco_url = "$au_GalleryUrl/packages/{0}/{1}" -f $global:Latest.PackageName, $package.RemoteVersion
+                if ( !$au_GalleryPackageRootUrl ) { 
+                    if ( $au_GalleryUrl ) { $au_GalleryPackageRootUrl = "$au_GalleryUrl/packages" }
```

Comment:
`$au_GalleryPackageRootUrl = if ($au_GalleryUrl) { "$au_GalleryUrl/packages" } else { 'https://chocolatey.org/packages' }`

Reply:
Looks  good to me

Reply:
Pushed your cleaner update.  Thanks.